JapaneseFilter/sections/specific.txt

```diff
@@ -2774,6 +2774,7 @@ pref.okayama.jp###top_sidebar1_banner
 [$path=/negishinouen]plaza.rakuten.co.jp###content-sticky div.side_lo a[href$="/ranking.html"] ~ a[target="_blank"]:not([href$="/ranking.html"])
 ||hbb.afl.rakuten.co.jp/hsb/$domain=plaza.rakuten.co.jp
 nikkansports.com##div[id^="adUnit"]
+@@||cxense.com^$domain=nikkansports.com
```

Comment:
Isn't `@@||cdn.cxense.com/cx.js$domain=nikkansports.com` enough?
Also wrong filter - check which one blocks the request.